README.md

```diff
@@ -1,2 +1,126 @@
+
+To allow users to explore these different modes of data access, we are also providing the PPDB (`association_{dataset}_{date}.db`) from which the sample alerts were generated as an [SQLite](https://www.sqlite.org) database.
+The SQLite database can be queried using SQL through a variety of interfaces.
+
+Note that in operations, users will query the PPDB using [ADQL](http://www.ivoa.net/documents/ADQL/) via a [TAP service](http://www.ivoa.net/documents/TAP/).
+
+### Software
+Users should install [lsst.alert.packet](https://github.com/lsst/alert_packet) to parse alerts and optionally [alert-stream-simulator](https://github.com/lsst-dm/alert-stream-simulator/) and [alert-stream](https://github.com/lsst-dm/alert_stream) if they wish to simulate a Kafka alert stream.
+
+## External References and Additional Resources
+
+The [Data Products Definition Document (DPDD)](http://ls.st/dpdd) provides a change-controlled guide to what scientific contents are planned for alerts and other Rubin Observatory data products.
+
+[LDM-612](http://ls.st/LDM-612) describes "Plans and Policies for LSST Alert Distribution," including a high-level overview of the relevant components, a summary of data rights issues, and extensive discussion of the role and selection of community alert brokers.
+
+[DMTN-093](https://dmtn-093.lsst.io/) presents the technical design of the alert distribution system.
+
+[DMTN-102](https://dmtn-102.lsst.io/) summarizes "Key Numbers" for alerts, estimating numbers, sizing, etc.
+
+[lsst.alert.packet](https://github.com/lsst/alert_packet) is a Python package that contains alert schemas as well as utilities for reading and writing Rubin alert packets.
+
+Community broker authors will be interested in simulating the Kafka alert stream itself.  [alert-stream-simulator](https://github.com/lsst-dm/alert-stream-simulator/) provides the relevant code; the overall design is discussed in 
+[DMTN-149](https://dmtn-149.lsst.io/).
+[alert-stream](https://github.com/lsst-dm/alert_stream) currently contains additional code for converting from the [Confluent Wire Format](https://dmtn-093.lsst.io/#management-and-evolution) used in the Kafka stream.
+
+## Data Sources
+
+### DECam-HiTS
+
+Sample alerts are provided from a subset of the DECam High Cadence Transient Search (HiTS; [Förster et al. 2016](https://ui.adsabs.harvard.edu/abs/2016ApJ...832..155F/abstract)).
+
+We process g-band data from three fields (`Blind15A_26`, `Blind15A_40`, and `Blind15A_42`) from the 2015 campaign and use coadded templates built from the matching fields in the 2014 campaign.  
+In total there are 5020 processed CCD-visits.
+
+Users interested in performing their own reductions can obtain these data [here](https://github.com/lsst/ap_verify_hits2015).
+The `ap_verify` package in the LSST Stack can be used to [process this dataset](https://pipelines.lsst.io/modules/lsst.ap.verify/running.html).
+
+## Limitations and Caveats
```

Comment:
This section is good and detailed. It would be nice if it could appear alongside the data permanently to, like on `lsst-web` alongside the `.avro` files, or perhaps linked to from a changelog, or something.

If a user goes and gets an old dataset, and then looks on the README and sees the latest notes, they may get confused.

Reply:
I was planning to include the appropriate version of the README alongside the avro files, as you suggest.